src/commands/General/latex.ts

```diff
@@ -63,6 +98,63 @@ export class LatexCommand extends HorizonCommand<typeof config> {
       replyTo = submit;
       equation = submit.fields.getTextInputValue('equation');
     }
-    await replyTo.reply(settings.apis.latex + encodeURIComponent(equation));
+
+    try {
+      await this._tryToGenerate(replyTo, equation, 0);
+    } catch (e) {
+      if (e instanceof EvalError) {
+        await replyTo.reply({
+          content: "Ça n'a pas l'air d'être une équation valide ! Le message d'erreur LaTeX est :\n"
+            + `\`${e.message}\``,
+          ephemeral: true,
+        });
+      } else {
+        await replyTo.reply({
+          content: "L'image de la formule n'a pas pu être générée.",
+          ephemeral: true,
+        });
+      }
+    }
+  }
+
+  private async _parseAndGenerate(equation: string): Promise<Buffer> {
+    const svg = this._mathJax.tex2svg(equation, { display: true }).children[0];
+
+    const svgText: string = this._mathJax.startup.adaptor.outerHTML(svg);
+
+    // MathJax ne renvoie pas d'erreur si l'équation est invalide, il faut donc vérifier nous-même
+    const svgError = findError(svg, 'data-mjx-error');
+    if (svgError)
+      throw new EvalError(svgError);
+
+    const pad = equation.length > 40 ? 45 : 4;
+    return sharp(Buffer.from(svgText))
+      .resize({
+        width: equation.length > 40 ? 1200 : Math.max(Math.floor(8 * equation.length), 64),
+      }) // Sinon l'image est trop petite et pixellisée
+      .extend({
+        top: pad,
+        bottom: pad,
+        left: pad,
+        right: pad,
```

Comment:
J'ai un peu expérimenté de mon côté, et tu penses quoi ce ça ? C'est un bon compromis je pense, qui scale en fonction de la taille de l'équation (avec un max à 1024). Ça rend pas trop mal, je te laisse tester

```suggestion
    return sharp(Buffer.from(svgText))
      .resize({ width: Math.min(Math.floor(6 * equation.length), 1024) })
      .extend({
        top: 10,
        bottom: 10,
        left: 10,
        right: 10,
```